home.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <header class="header">
+
+        <nav class ="desktopnav">
+            <div class="space"></div>
+            <ul class="socialmedia">
+                <li><a href="#"><i class=" fa fa-brands fa-facebook-f"></i></a></li>
+                <li><a href="#"><i class=" fa fa-brands fa-twitter"></i></a></li>
```

Comment:
- [ ] Please my comment in the video I recorded for you 😉 

https://drive.google.com/file/d/1Erc6WADlwYn6CHjYivPusTo4vnQzixzD/view?usp=sharing